include/crm/common/output.h

```diff
@@ -66,13 +66,15 @@ typedef enum {
     pcmk_show_rsc_only      = 1 << 8,
     pcmk_show_failed_detail = 1 << 9,
     pcmk_show_feature_set   = 1 << 10,
+    pcmk_show_description   = 1 << 11,
 } pcmk_show_opt_e;
 
-#define pcmk_show_details   (pcmk_show_clone_detail     \
-                             | pcmk_show_node_id        \
-                             | pcmk_show_implicit_rscs  \
-                             | pcmk_show_failed_detail  \
-                             | pcmk_show_feature_set)
+#define pcmk_show_details   ((pcmk_show_clone_detail)     \
+                             | (pcmk_show_node_id)        \
+                             | (pcmk_show_implicit_rscs)  \
+                             | (pcmk_show_failed_detail)  \
+                             | (pcmk_show_feature_set)    \
+                             | (pcmk_show_description))
```

Comment:
The individual names don't need parentheses

Reply:
To clarify a bit, parentheses are helpful for macro arguments used in the macro body, but defined constants are usually not a problem because the best practice there is to define them with parentheses if it could matter. However if you're not sure how the constant is defined, parentheses are indeed safe, so feel free to leave this as is if you haven't already changed it.